src/components/Table/index.stories.js

```diff
@@ -8,7 +8,7 @@ import Icon from '../Icon';
 import Table from '.';
 import Text from '../Text';
 
-const data = [...new Array(50)].map(() => ({
+const data = [...new Array(7)].map(() => ({
```

Comment:
That's a bit easier to read on the Storybook

Reply:
I didn't follow, can you elaborate?

Reply:
I mean I don't wanna scroll down 50 rows to see the next example :smile:

Reply:
ah lol yeah 😄